jobs/release.Jenkinsfile

```diff
@@ -273,7 +273,12 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                 }]}
             }
         }
-
+        stage('Fork Garbage Collection') {
```

Comment:
Feels a bit odd to have this stage in the middle. I think let's put it last within this block.